.github/workflows/rotki_ci.yml

```diff
@@ -0,0 +1,265 @@
+          restore-keys: |
+            ${{ runner.os }}-cypress-
+      - name: Install dependencies
+        working-directory: ./electron-app
+        run: npm ci
+      - name: Lint
+        working-directory: ./electron-app
+        run: npm run lint:check
+      - name: Build
+        working-directory: ./electron-app
+        run: npm run build
+      - name: Run unit tests
+        working-directory: ./electron-app
+        run: npm run test:unit
+  lint-backend:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.7]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: Install dependencies
+        run: |
+          pip install -r requirements_lint.txt
+          pip install -e .
+          git rev-parse HEAD
+      - name: Lint
+        run: make lint
+
+  test-backend:
+    needs: lint-backend
+    env:
+      CI: true
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.7]
+        node-version: [12.x]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download Geth
+        run: |
+          .travis/download_geth.sh ${{ matrix.os }}
+          echo "::add-path::$HOME/.bin"
+        env:
+          GETH_URL_LINUX: https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.8.13-225171a4.tar.gz
+          GETH_URL_MACOS: https://gethstore.blob.core.windows.net/builds/geth-darwin-amd64-1.8.13-225171a4.tar.gz
+          GETH_VERSION: 1.8.13
+      - name: Setup SQLCipher
+        run: |
+          if [ ${{ matrix.os }} == 'ubuntu-latest' ];
+          then
+            sudo apt-get install libxml2-utils
+            ./install_deps.sh
+            sudo ldconfig
+          fi
+          if [ ${{ matrix.os }} == 'macos-latest' ];
+          then
+            brew install sqlcipher
+          fi
+      - name: Set up python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - uses: actions/cache@v1
+        if: startsWith(runner.os, 'Linux')
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      - uses: actions/cache@v1
+        if: startsWith(runner.os, 'macOS')
+        with:
+          path: ~/Library/Caches/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          pip install --upgrade "pip<19.0.0" wheel
+          pip install pytest-travis-fold codecov pytest-cov
+          pip install -r requirements_dev.txt
+          pip install -e .
+      - name: Run Test
+        run: |
+          COVERAGE_ARGS='--cov=./ --travis-fold=always'
```

Comment:
In Travis we were invoking some scripts. Can the same not happen here?

Reply:
I will have to modify the script to make it compatible with GitHub Actions due to the following
https://github.com/rotki/rotki/blob/9f76312ad7552e2d35b6342e2ba3a3f13880fc98/.travis/run_tests.sh#L7

Reply:
I see. No need to do so. Just wanted to know if the syntax allowed it